chainercv/links/connection/conv_2d_bn_activ.py

```diff
@@ -57,22 +69,36 @@ class Conv2DBNActiv(chainer.Chain):
             :func:`chainer.functions.relu`.
         bn_kwargs (dict): Keyword arguments passed to initialize
             :class:`chainer.links.BatchNormalization`.
+        comm (:class:`~chainermn.communicators.CommunicatorBase):
+            If a ChainerMN communicator is given,
+            :obj:`~chainermn.links.MultiNodeBatchNormalization` will be used
+            for the batch normalization. If :obj:`None`,
+            :obj:`~chainer.links.BatchNormalization` will be used.
```

Comment:
I prefer to take  `comm` as an element of `bn_kwargs` because `comm` is used only for batchnorm.
If `'comm' in bn_kwargs`, it use `MultiNodeBatchNormalization`.

Reply:
OK, I changed it